### ISSUE TITLE
Add translation option for emoji categories 

### ISSUE THREAD

**Issue by BigMinnie** [clarkwinkelmann/flarum-ext-emojionearea#18]
Add a option to translate categories:

- Recent
- Smileys & People
- Animals & Nature
- Food & Drink
- Activity
- Travel & Places
- Objects
- Symbols
- Flags

![Screenshot_1](https://user-images.githubusercontent.com/32235570/81736594-1b775780-9497-11ea-8424-e212cbce1b5c.png)

**Comment by clarkwinkelmann** [comment]
Yes that would be great... Unfortunately the EmojiOneArea library does not make it easy to change the hard-coded English text.

If I find an easy solution I'll implement it but my hopes are low on this one.

**Comment by rg190** [comment]
Yes the translation would be great.

**Comment by clarkwinkelmann** [comment]
Categories can now be translated, starting with version 1.1.0
